### ISSUE TITLE
UNET Training

### ISSUE THREAD

**Issue by lewismcnish** [OirthirSAT/image-processing-pipeline#16]
## Overview

Train the Neural Network on more data, and export and save the training data parameters, while doing so, write a quick document on how this saving and reusing of a parameter file is done for the use of other team members. If more training data is required, discuss with the team and we will manually train/outsource training of images.
Run an analysis on the number of images used in training vs the accuracy of the Neural Network this can be done by eye or statistically depending on time spent, share this analysis with the team.

## Requirements

(In order of importance)

1. Train the Neural network on more data
2. Export and Save training parameters
3. Write a document on training parameters process
4. Discuss the training of more data
5. Analysis of the number of training images vs accuracy of NN

## Definition of Done

1. Training parameters file exported and contains more training than current version (Include number of training images in file name)
2. New document on how this training parameters file works
3. (EXTRA)-Analysis of NN presented to team upon completion of more training

## Resources

Training parameter Document can be written as a markdown and included in the UNET folder within the repository or as a word document in teams file system (Speak to Lewis/Joe about the formatting and location of document if using teams)
